### ISSUE TITLE
Customization breaks semantics in "Getting started"

### ISSUE THREAD

**Issue by WeirdKeksButtonK** [holoviz/holoviews#4414]
```
python==3.7.7
bokeh==2.0.2
holoviews==1.13.2
jupyterlab==2.1.2
```

_This is not a bug in the direct sense_, but I feel like some comments are missing in the tutorial.

I am studying the [Customization](http://holoviews.org/getting_started/Customization.html) section of "Getting Started". At the begging of the section the curve and the spikes "share" the X axis, i.e. moving one plot makes another also move. However, in the end, after `redim` was used, this property was gone. I am confused at this point and I am not sure how exactly I should interpret this behavior. Probably, some additional comments in the tutorial could help (or, at least here in the issue 😄 ).

It can be reproduced with the notebook from the `holoviews --install-examples`.

**Comment by jbednar** [comment]
Good point! You can restore the shared axes by redefining _both_ x axes the same way:

```
xlabel = 'Time in milliseconds (10⁻³ seconds)'
spikes = spikes.redim.label(milliseconds=xlabel)
curve  = curve.redim.label(Hertz='Frequency (Hz)', milliseconds=xlabel)
(curve + spikes).select(milliseconds=(2000,4000)).cols(1)
```

Dimensions are considered the same for linking purposes if they share the same name and label; see 
http://holoviews.org/reference_manual/holoviews.core.html#holoviews.core.Dimension .

We should update this example to make sure that the axes stay linked!
